### PR TITLE
Add Slack log handler

### PR DESCRIPTION
**Pull request by andela-ookoro**
#### What does this PR do?
Add slack log handler to the application.
#### Description of Task to be completed?
Add SLACK_URL to project setting
Add the slack handler to the application's loggers.
#### How should this be manually tested?
* export the variables HTOOLS_SLACK_URL.
* log an error or warning message.
#### Screenshots
htoolsslacklog

### CODE REVIEW

**Comment by DavidLemayian** [comment]
For easy readability, let's have the message posted to Slack look like New Relic's:

slack1

**Comment by DavidLemayian** [comment]
Could we switch this out with https://github.com/CodeForAfricaLabs/python-slack-logger ?

**Comment by DavidLemayian** [comment]
Does this require a `requirements.txt` update as well?

**Comment by andela-ookoro** [comment]
Yea, I have updated the PR with the latest package requirements.

File: requirements.txt [hunk 1]
```diff
@@ -10,11 +10,17 @@ idna==2.6
...
 MarkupSafe==1.0
+nested-lookup==0.1.3
+prompt-toolkit==1.0.15
 PyJWT==1.5.3
+PySocks==1.6.7
 pytz==2017.2
 requests==2.18.4
 requests-aws4auth==0.9
 six==1.11.0
+slack-logger==0.2.0
```
Comment (DavidLemayian) [inline requirements.txt:21]: I don't think this is the same `slack-logger` that you've worked on here - https://github.com/CodeForAfricaLabs/python-slack-logger

**Comment by andela-ookoro** [comment]
yea, I just reinstall the package now but it did not reflect the changes.

**Pull request closed without merging**